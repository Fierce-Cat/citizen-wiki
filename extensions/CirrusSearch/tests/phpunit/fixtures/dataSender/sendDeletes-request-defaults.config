{
  "config": {
      "CirrusSearchIndexBaseName": "myfixturewiki",
      "CirrusSearchUpdateShardTimeout": "1ms"
  },
  "indexSuffix": "content",
  "ids": [ "1", "2" , "3" ]
}
