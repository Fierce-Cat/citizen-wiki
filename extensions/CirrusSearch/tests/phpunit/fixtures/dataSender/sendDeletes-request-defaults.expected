{
    "path": "_bulk",
    "method": "POST",
    "data": [
        {
            "delete": {
                "_id": "1",
                "_index": "myfixturewiki_content"
            }
        },
        {
            "delete": {
                "_id": "2",
                "_index": "myfixturewiki_content"
            }
        },
        {
            "delete": {
                "_id": "3",
                "_index": "myfixturewiki_content"
            }
        }
    ],
    "params": [],
    "contentType": "application\/x-ndjson"
}